### PR TITLE
New Flow.to_dict implementation.

### PR DESCRIPTION
**Pull request by gtopper**
Via @yaronha.

### CODE REVIEW

File: storey/flow.py [hunk 1]
```diff
@@ -26,39 +27,56 @@ def __init__(self, recovery_step=None, termination_result_fn=lambda x, y: x if x
         self.logger = getattr(self.context, 'logger', None) if self.context else None
 
         self._kwargs = kwargs
-        self._full_event = kwargs.pop('full_event', False)
```
Comment (yaronha) [inline storey/flow.py:29]: why not pop (no need to keep it in the kwargs as well)

Comment (gtopper) [reply storey/flow.py:29]: It's only in kwargs. The new `to_dict` doesn't explicitly refer to `_full_event`.

Comment (gtopper) [reply storey/flow.py:29]: Updated

**Comment by yaronha** [comment]
@gtopper LGTM

**Pull request merged**